src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IUserReleaseInviteRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces
+{
+    public interface IUserReleaseInviteRepository
+    {
+        Task<Either<ActionResult, Unit>> Create(Guid releaseId, string email, ReleaseRole releaseRole,
+            bool emailSent, Guid createdById, bool accepted = false);
+
+        Task<Either<ActionResult, Unit>> CreateManyIfNotExists(List<Guid> releaseIds, string email,
+            ReleaseRole releaseRole, bool emailSent, Guid createdById, bool accepted = false);
```

Comment:
Would format these so that they're nicely formatted as multi-line args. No need to have them crammed onto two lines, particularly as it makes seeing diffs in the lines a little harder later on.